docs/tutorials/hello-world.md

```diff
@@ -65,7 +62,7 @@ Now let’s look at what is in the smart contract.
 npx scrypt-cli compile
 ```
 
-This command will generate a contract artifact file at `artifacts\helloworld.json`.
+This command will generate a contract artifact file at `/artifacts/helloworld.json`.
```

Comment:
made slashes consistent with OSX. 

i know windows handles these differently, but the rest of the docs use the `/`